Code/Geometry/UniformGrid3D.cpp

```diff
@@ -222,33 +212,31 @@ void UniformGrid3D::setSphereOccupancy(const Point3D &center, double radius,
       bgRad + nLayers * gStepSize;  // largest radius in grid coords
   double gRad2 = gRadius * gRadius;
   double bgRad2 = bgRad * bgRad;
-  int i, j, k;
   double dx, dy, dz, d, d2, dy2z2, dz2;
-  int xmin, xmax, ymin, ymax, zmin, zmax;
-  xmax = (int)floor(gPt.x + gRadius);
-  xmin = (int)ceil(gPt.x - gRadius);
-  ymax = (int)floor(gPt.y + gRadius);
-  ymin = (int)ceil(gPt.y - gRadius);
-  zmax = (int)floor(gPt.z + gRadius);
-  zmin = (int)ceil(gPt.z - gRadius);
+  int xmax = (int)floor(gPt.x + gRadius);
```

Comment:
while you're making changes in this file, please convert all of these C-style casts to `static_casts`